examples/default/machinedeployment/machinedeployment.yaml

```diff
@@ -55,7 +55,7 @@ spec:
           name: "{{ ds.meta_data.hostname }}"
           criSocket: "/var/run/containerd/containerd.sock"
           kubeletExtraArgs:
-            cloud-provider: vsphere
+            cloud-provider: external
```

Comment:
Hi @andrewsykim,

Does the worker node even need to set anything on the Kubelet's args now that the CPI is external?

Reply:
the `--cloud-provider=external` is still required in the kubelet for the external provider to know the node requires registration

Reply:
Got it.